sys-libs/zlib/zlib-1.3.1.recipe

```diff
@@ -12,8 +12,8 @@ reduced, if necessary, at some cost in compression."
 HOMEPAGE="http://www.zlib.net/"
 COPYRIGHT="1995-2017 Jean-loup Gailly and Mark Adler"
 LICENSE="Zlib"
-REVISION="3"
-SOURCE_URI="https://zlib.net/fossils/zlib-$portVersion.tar.gz"
+REVISION="4"
+SOURCE_URI="https://github.com/madler/zlib/releases/download/v$portVersion/zlib-$portVersion.tar.gz"
```

Comment:
We could also keep both URLs to have an additional mirror.

Reply:
I think zlib.net removes old archives, not really a valid mirror.

Reply:
The fossils directory contains all the older tarballs: https://zlib.net/fossils/

Just the main download on the homepage gets replaced with each update.

Reply:
@jmairboeck is right, fossils seems to keep everything mirrored. My suggestion was based on this commit at [Arch](https://gitlab.archlinux.org/archlinux/packaging/packages/zlib/-/commit/a15e0ee1f540cdb2f2c4fd57476c0116358e20df).

Reply:
> > The fossils directory contains all the older tarballs: https://zlib.net/fossils/
> 
> Yes, "fossils" seems to be an apt name indeed :-)
> 
> In any case @jmairboeck, sorry for not reacting sooner to this. I wasn't really sure what you meant by "keep both URLs to have an additional mirror". Did you meant leaving the old SOURCE_URI as a comment, or something else?

SOURCE_URI is a multi-value variable and supports multiple URLs (see https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#source_uri). Granted that this isn't used very much.

Of course we can also just keep the Github URL, that isn't going away either ;)